### PR TITLE
First commit:

### PR DESCRIPTION
**Pull request by rodrigopavezi**
Project structure
Start page
Presentation list

https://editor-stage-0.risevision.com/

@alex-deaconu sorry for the long commit. It is just the first one. I have also done the presentation list and the start page as it was easily copied from schedules. Please review.

@ezequielc FYI.

Cheers

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "rv-schedules-app",
```
Comment (alex-deaconu) [inline bower.json]: @rodrigopavezi this needs to be changed. Thanks.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    <link rel="chrome-webstore-item" href="https://chrome.google.com/webstore/detail/mfpgpdablffhbfofnhlpgmokokbahooi">
+
+    <link rel="canonical" href="https://editor.risevision.com/" itemprop="url">
+    <!-- *** Editor App Meta *** -->
+    <title itemprop='name'>Rise Vision Editor | Digital Signage Network Management</title>
+    <meta name="description" content="Schedule delivery of HTML content to your digital signage displays, anywhere, anytime." />
+
+    <!-- Schema.org markup for Google+ -->
+    <meta itemprop="name" content="Rise Vision Editor">
+    <meta itemprop="description" content="Schedule delivery of HTML content to your digital signage displays, anywhere, anytime.">
```
Comment (alex-deaconu) [inline index.html]: @rodrigopavezi I believe this needs to be updated. Should say: "Create and manage HTML content for your digital signage displays, anywhere, anytime.". Same with the descriptions below. Thanks!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    <meta itemprop="image" content="https://s3.amazonaws.com/Rise-Images/landing-page/schedule-image.png">
```
Comment (alex-deaconu) [inline index.html]: @rodrigopavezi needs to be updated. Thanks.

File: js/controllers/ctr-app.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+angular.module('risevision.editorApp.controllers')
+  .controller('AppCtrl', ['$scope', '$location',
+    function ($scope, $rootScope, $location) {
+      $scope.navOptions = [{
+        title: 'Editor',
```
Comment (alex-deaconu) [inline js/controllers/ctr-app.js]: @rodrigopavezi I'm wondering if this should say `Presentations`. Thoughts? Thanks.

File: js/controllers/ctr-presentation-list.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+        $scope.load();
+      };
+
+      $scope.doSearch = function () {
+        $scope.presentations.clear();
+
+        $scope.load();
+      };
+
+      $scope.select = function (presentationId, presentationName) {
```
Comment (alex-deaconu) [inline js/controllers/ctr-presentation-list.js]: @rodrigopavezi I don't think this is needed. Thanks.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "rv-schedules-app",
```
Comment (alex-deaconu) [inline package.json]: @rodrigopavezi this needs to be updated to `editor` everywhere. Thanks.

**Comment by alex-deaconu** [comment]
@rodrigopavezi looks good :+1: . Just a couple of missed "Schedules" references that need to be updated. Thanks.

File: js/controllers/ctr-app.js (hunk 4 above)
Comment (rodrigopavezi) [inline js/controllers/ctr-app.js]: @alanclayton makes sense. I will change.

File: index.html (hunk 2 above)
Comment (rodrigopavezi) [inline index.html]: @alex-deaconu I was not going to work on metadata now. But why not. So, I will make the changes. cheers

Comment (alex-deaconu) [inline index.html]: @rodrigopavezi up to you. I just figured if we get everything cleaned up we don't have to return to it later. Thanks.

**Comment by rodrigopavezi** [comment]
@alex-deaconu fixed all the commented issue. cheers

**Pull request merged**